irl_robot_command: "movej"
irl_robot_accl: 0.8
irl_robot_vel: 0.8
irl_robot_com_time: 2
ur5:
  control_params:
    p_gain: 2.0
    i_gain: 1.0
    d_gain: 0.0
    m_max_velocity: 1.57
    m_i_max: 0.5
    m_i_min: -0.5
    m_max_accel: 3.14
  state_size: 1108  

# ur5:
#   control_params:
#     p_gain: 3.0
#     i_gain: 0.05
#     d_gain: 0.0
#     m_max_velocity: 1.57
#     m_i_max: 1.57
#     m_i_min: -1.57
#     m_max_accel: 1.57